test/passing/comments.ml

```diff
@@ -130,21 +130,27 @@ a : int [@default a]
 (* comment *)
 ;  b : flag }
 
-;;
-match (* comment about entire pattern matching *)
-      e with k -> z
```

Comment:
Could you add some tests where the comment comes before `match` as well, to check that their relative positions are preserved.